include/haproxy/global-t.h

```diff
@@ -85,6 +85,7 @@
 #define GTUNE_LISTENER_MQ_OPT    (1<<28)
 #define GTUNE_LISTENER_MQ_ANY    (GTUNE_LISTENER_MQ_FAIR | GTUNE_LISTENER_MQ_OPT)
 #define GTUNE_QUIC_CC_HYSTART    (1<<29)
+#define GTUNE_MPTCP      		 (1<<30)
```

Comment:
the alignment is not OK because you use tabs apparently

Reply:
Mmh, I just noticed you fixed that in the 3rd commit. Please next time squash such commits, and do 'rebase' on top of master, no 'merge' with master which creates an extra commit we don't need to review.